com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -495,6 +502,9 @@ modules:
         ln -s ../share dist/share
         ln -s ../version dist/version
       - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - mkdir -p /app/utils /app/share/vulkan
+      - ln -srv /app/{utils/,}share/vulkan/explicit_layer.d
+      - ln -srv /app/{utils/,}share/vulkan/implicit_layer.d
```

Comment:
I'd rather put these to the `metadata` module and keep `proton` module installing only Proton itself.